src/coreComponents/constitutive/contact/CoulombFriction.hpp

```diff
@@ -265,6 +265,8 @@ inline void CoulombFrictionUpdates::computeShearTraction( localIndex const k,
       real64 dLimitTau_dNormalTraction;
       real64 const limitTau = computeLimitTangentialTractionNorm( tractionVector[0],
                                                                   dLimitTau_dNormalTraction );
+      // dLimitTau_dNormalTraction from the function above has a wrong sign, flip it
+      dLimitTau_dNormalTraction = -dLimitTau_dNormalTraction;
```

Comment:
why not fixing it inside the function?

Reply:
the functions is being used also by `SolidMechanicsLagrangeContact`, just limiting changes to EDFM here

Reply:
but the sign should not change. I mean I would expect that derivative to be the same